projects/js-packages/charts/src/components/legend/stories/index.stories.tsx

```diff
@@ -197,7 +222,7 @@ export const WithBarChart: Story = {
 	parameters: {
 		docs: {
 			description: {
-				story: 'Legend used with BarChart data, positioned vertically beside the chart.',
+				story: 'Legend used with BarChart using the composition API, positioned beside the chart.',
```

Comment:
This is positioned below

![Screenshot 2025-07-29 at 10 39 15 AM](https://github.com/user-attachments/assets/add038b1-4f19-4c56-9589-ab66b6449555)

Reply:
It sure is! 🙈